src/components/tops/ActionPanel.vue

```diff
@@ -1830,29 +1823,22 @@ div.assignation {
 
 .status-item {
   align-items: center;
-  border: 2px solid $light-grey;
+  border: 2px solid var(--text);
   border-radius: 15px;
+  color: var(--text);
   font-weight: bold;
   display: flex;
   font-size: 0.7em;
   justify-content: center;
   height: 100%;
   margin-left: 1em;
   margin-top: -1em;
+  padding-top: 0.3em;
   text-transform: uppercase;
 
-  &:hover {
-    border: 2px solid var(--text);
-  }
-
   &.active {
-    border: 2px solid $light-green;
-    color: $light-green;
-
-    &:hover {
-      border: 2px solid $light-green;
-      color: $light-green;
-    }
+    border: 2px solid var(--text-selected);
```

Comment:
Prefer to use `border-color`

Reply:
No, `border-color` leads often to unexpected behaviour.